RELEASE_NOTES.md

```diff
@@ -2,6 +2,170 @@
 
 <!-- newest release at the top please) -->
 
+# Prod 2019/04/02
+
+### Ingest
```

Comment:
Nit: Is skipping level two heading intentional?

Reply:
Fixed. Thank you for catching that!!!